extras/local-omi-bt/send_to_adv.py

```
@@ -35,7 +44,7 @@ async def get_jwt_token(username: str, password: str) -> Optional[str]:
     try:
         logger.info(f"Authenticating with backend as: {username}")
 
-        async with httpx.AsyncClient(timeout=10.0) as client:
+        async with httpx.AsyncClient(timeout=10.0, verify=False) as client:
```

Ruff (0.14.13)

Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)